src/components/bulkConcepts/component/PreviewCard.jsx

```diff
@@ -61,6 +62,7 @@ const PreviewCard = ({
             </small>
           </p>
           <div className="pop-up-description rounded">{display_name}</div>
+          <CardBody title="Synonym" body={names ? names[1].name : 'N/A'} />
```

Comment:
How many names would you expect?

Reply:
@dkayiwa the data that's returned includes key `names` that has an array of objects. The object at index 1 contains the other name for the concept or the synonym.

Reply:
What happens in cases of more than one synonym?

Reply:
@dkayiwa I have changed the implementation to take that into consideration.

Reply:
Does the screenshot reflect it?

Reply:
@dkayiwa let me change the screenshot. It had skipped my mind.

Reply:
@dkayiwa I have added the screenshot